src/science/soilsnow/cbl_snowAccum.F90

```diff
@@ -47,6 +47,8 @@ SUBROUTINE snow_accum ( dels,  canopy, met, ssnow, soil )
 
              ssnow%tgg(i,1) = ssnow%tgg(i,1) + canopy%precis(i) * CHLF               &
                   / ( REAL( ssnow%gammzz(i,1) ) + Ccswat *canopy%precis(i) )
+             ssnow%dtmlt(i,1) = ssnow%dtmlt(i,1) + canopy%precis(i) * CHLF           &
```

Comment:
these are new terms from AM3 development. they are the counterpoint to the melt terms to `%dtmlt` and come about by the freezing of liquid precip onto frozen ground or snow pack. 

good that they've been picked up - but not that surprising given they sit in a different part of the code base to the rest of the ice fixes